docs/bokeh/source/docs/dev_guide/setup.rst

```diff
@@ -264,20 +264,7 @@ different local version instead, set the ``BOKEHJS_ACTION`` environment variable
 
 .. _contributor_guide_setup_sample_data:
 
-7. Download sample data
------------------------
-
-Several tests and examples require Bokeh's sample data to be available on your
-hard drive. After :ref:`installing <contributor_guide_setup_install_locally>`
-Bokeh, use the following command to download and install the data:
-
-.. code-block:: sh
-
-    pip install bokeh_sampledata
-
-.. _contributor_guide_setup_environment_variables:
```

Comment:
You need to keep this and remove the earlier ancho (on line 265).

Reply:
Thank you for this. I'll reflect the changes shortly.